doc/source/api.rst

```diff
@@ -579,6 +571,8 @@ To create a Series of dtype ``category``, use ``cat = s.astype("category")``.
 The following two ``Categorical`` constructors are considered API but should only be used when
 adding ordering information or special categories is need at creation time of the categorical data:
 
+.. currentmodule:: pandas.core.categorical
```

Comment:
BTW, this does not need to be `pandas.core.categorical`, the user API is just `pandas.Categorical`